### PR TITLE
DOC: Add example for calculating activity

### PR DESCRIPTION
**Pull request by bocklund**

### CODE REVIEW

File: docs/examples/PlotActivity.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    ref_eq = equilibrium(dbf, comps, phases, {v.P: 101325, v.T: 1023, v.X('ZN'): 1})
+
+Calculate the equilibria
+------------------------
+
+Do the calculation over the compoisition range
+
+.. code:: ipython3
+
+    eq = equilibrium(dbf, comps, phases, {v.P: 1013325, v.T: 1023, v.X('ZN'): (0, 1, 0.005)})
```
Comment (richardotis) [inline docs/examples/PlotActivity.rst]: That's a pretty fine step size. Is performance acceptable?

File: docs/examples/PlotActivity.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    chempot = eq.MU.sel(component='ZN').squeeze()
+    
+    acr_zn = np.exp((chempot - chempot_ref)/(8.315*1023))
+
+Experimental activity results
+-----------------------------
+
+For comparision, we'll add experimental results.
+
+Activties are digtized from Fig 18 in Yazawa, A., & Lee, Y. K. (1970).
```
Comment (richardotis) [inline docs/examples/PlotActivity.rst]: spelling (two places)

File: docs/examples/PlotActivity.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Thermodynamic Studies of the Liquid Aluminum Alloy Systems. Transactions
+of the Japan Institute of Metals, 11(6), 411–418.
+http://doi.org/10.2320/matertrans1960.11.411
+
+The measurements at at 1073 K (the same temperature our calculations
```
Comment (richardotis) [inline docs/examples/PlotActivity.rst]: "The experimental measurements are taken at 1073 K (same as our calculation temperature) and the reference state is pure Zn at that same temperature."

File: docs/examples/PlotActivity.rst [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Set up the database
+~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    from pycalphad import Database, equilibrium, variables as v
+    import numpy as np
+    
+    dbf = Database('alzn_mey.tdb')
```
Comment (richardotis) [inline docs/examples/PlotActivity.rst]: Database needs a citation

File: docs/examples/PlotActivity.rst (hunk 1 above)
Comment (bocklund) [reply docs/examples/PlotActivity.rst]: Yeah it's pretty fast. Takes about 3s on my machine (JIT was in previous step).

**Comment by bocklund** [comment]
Thanks for the good feedback. I need a spellcheck plugin for Jupyter Notebooks.

**Pull request merged**